docroot/modules/custom/webform_custom_submissions/src/TravelformHandler.php

```diff
@@ -0,0 +1,486 @@
+      $postResponse = $this->sendPOSTData($postData);
+
+      //Uncomment to show full repsonse in broswer
+      //drupal_set_message(json_decode(var_export($postResponse, true)));
+      //var_export($_POST,true);
+
+      //dpm($_POST);
+      //drupal_set_message($postData, 'error');
+      //drupal_set_message($postResponse, 'error');
+      $decoded = json_decode($postResponse, TRUE);
+      if (isset($decoded['id'])) {
+        $page = explode('/', \Drupal::service('path.current')->getPath());
+        \Drupal::logger('Travel Services Response ' . $page[sizeof($page) - 1])->notice($postResponse);
+        $issueId = $this->getIssueId($postResponse);
+
+        $filesUploaded = $this->attachFiles($issueId, $form_state);
+        $this->parseIssueResponse($postResponse, $filesUploaded, $_POST['formTitle'], $postData, $_FILES);
+      } elseif (isset($decoded['errorMessages'])) {
+        \Drupal::logger('Travel Services Error')->notice($postResponse);
+        drupal_set_message(t('There was an error processing your request. Code-0001'), 'error');
+        mail('zerihun.tegegn@cgifederal.com', 'TravelServices Error', $postResponse);
+      } else {
+        drupal_set_message(t('There was an error processing your request. Code-0002'), 'error');
+        \Drupal::logger('Travel Services Error')->notice('Unidentified Error: JIRA Response = ' . $postResponse);
+      }
+    } catch (Exception $e) {
+      \Drupal::logger('Travel Services Exception')->notice($e->getMessage());
+      drupal_set_message(t('Unable to process request at this time, please try again later.'), 'error');
+      //drupal_set_message($e->getMessage());
+    }
+  }//end submitToJira
+
+//Compiles POST data and returns the data formatted as JSON
+  protected function compilePOSTData() {
+
+    $dropDowns = array(
+      'customfield_10093',
+      'customfield_10098',
+      'customfield_10112',
+      'customfield_10120',
+      'customfield_10140',
+      'customfield_10190',
+      'customfield_10191',
+      'customfield_10193',
+      'customfield_10199',
+      'customfield_10246',
+      'customfield_10262',
+      'customfield_10263',
+      'customfield_10269',
+      'customfield_10277',
+      'customfield_10294',
+      'customfield_10297',
+      'customfield_10302',
+      'customfield_10304',
+      'customfield_10315',
+      'customfield_10316',
+      'customfield_10318',
+      'customfield_10320',
+      'customfield_10322',
+      'customfield_10324',
+      'customfield_10423',
+      'customfield_10424',
+      'customfield_10426',
+      'customfield_10427',
+      'customfield_10428',
+      'customfield_10431',
+      'customfield_10435',
+      'customfield_10501',
+      'customfield_10922',
+      'customfield_11424',
+      'customfield_11428',
+      'customfield_11826',
+      'customfield_11827',
+    );
+
+    $data = array('fields' => array());
+
+    //Add POST variables to the array
+    foreach ($_POST as $key => $val) {
```

Comment:
I don't think webform handler actually uses $_POST, but instead there is class method. This is a good guide in how to create a webform handler: https://www.lilengine.co/articles/webform-handler